app/services/activity/import.rb

```diff
@@ -131,7 +146,21 @@ def initialize(row:, uploader:, partner_organisation:, report:, is_oda:)
       def update
         return unless @activity && @errors.empty?
 
-        @activity.assign_attributes(@converter.to_h)
+        attributes = @converter.to_h
+
+        if @is_oda == false
+          level = @activity.programme? ? :level_b : :level_c_d
```

Comment:
Sorry - I didn't realise we'd need to convert the level into a symbol here to then be checked in the method - I think it might be cleaner to pass in the `activity` itself here and call `.programme?` inside `unexpected_non_oda_attribute_errors` here.

Reply:
I was conflicted about whether to pass it in as the symbol or `@activity.level` - I went with the symbol to keep it consistent with `filtered_csv_column_headings`, but it might be cleaner to pass the `@activity.level` and do the conversion inside (like before, but without the whole activity), or yeah, pass the whole activity and use the `programme?` method. Any preference?

Reply:
I think passing in and use the `programme?` would be my preference, but up to you - apologies again for flip-flopping on this!